### PR TITLE
Replacing YT embeds with Stream embeds

### PR DESCRIPTION
**Pull request by jason-cf**
closes pcx-10549

### CODE REVIEW

File: content/fundamentals/_partials/_what-is-cloudflare.md [hunk 1]
```diff
@@ -5,14 +5,15 @@ _build:
...
+    <iframe
+        src="https://customer-1mwganm1ma0xgnmj.cloudflarestream.com/bdf5733bed92deded731efee29b2346f/iframe?letterboxColor=transparent&preload=true&poster=https%3A%2F%2Fcustomer-1mwganm1ma0xgnmj.cloudflarestream.com%2Fbdf5733bed92deded731efee29b2346f%2Fthumbnails%2Fthumbnail.jpg%3Ftime%3D70s"
+        style="border: none; position: absolute; top: 0; left: 0; height: 100%; width: 100%;"
+        allow="accelerometer; gyroscope; autoplay; encrypted-media; picture-in-picture;"
+        allowfullscreen="true">
+    </iframe>
+</div>
 
-![Cloudflare sits in between requests and your origin server.](/images/fundamentals/get-started/website-with-cloudflare.svg)
+Cloudflare is one of the world’s largest [connectivity cloud networks](https://blog.cloudflare.com/welcome-to-connectivity-cloud). Today, businesses, non-profits, bloggers, and anyone with an Internet presence boast faster, more secure websites and apps thanks to Cloudflare.
```
Comment (RebeccaTamachiro) [inline content/fundamentals/_partials/_what-is-cloudflare.md]:
```suggestion
Cloudflare is one of the world’s largest [connectivity cloud networks](https://blog.cloudflare.com/welcome-to-connectivity-cloud). Today, anyone with an Internet presence can have faster and more secure websites and applications thanks to Cloudflare. This includes bloggers, businesses, and even non-profits.
```

Comment (RebeccaTamachiro) [reply content/fundamentals/_partials/_what-is-cloudflare.md]: Just a suggestion to make the sentence more readable.

File: content/fundamentals/concepts/what-is-cloudflare.md [hunk 2]
```diff
@@ -6,4 +6,35 @@ weight: 1
 
 # What is Cloudflare?
 
-{{<render file="_what-is-cloudflare.md" productFolder="fundamentals">}}
\ No newline at end of file
+{{<render file="_what-is-cloudflare.md" productFolder="fundamentals">}}
+
+## What is a connectivity cloud?
+
+A connectivity cloud is a new approach for delivering the many services companies need to secure and connect their digital environment. It’s a unified, intelligent, platform of programmable cloud-native services that enable any-to-any connectivity between all networks (enterprise and Internet), cloud environments, applications and users. It includes a huge array of security, performance, and developer services — not with an eye to replace everything everywhere, but with the ability to fit in wherever needed and consolidate many critical services onto a single platform.
```
Comment (RebeccaTamachiro) [inline content/fundamentals/concepts/what-is-cloudflare.md]:
```suggestion
A connectivity cloud is a new approach for delivering the many services companies need to secure and connect their digital environment. 

As a unified and intelligent platform of programmable cloud-native services, a connectivity cloud enables any-to-any connectivity between all networks (enterprise and Internet), cloud environments, applications and users. 

Cloudflare's connectivity cloud includes a huge array of security, performance, and developer services — not with an eye to replace everything everywhere, but with the ability to fit in wherever needed and consolidate many critical services onto a single platform.
```

**Review by RebeccaTamachiro — APPROVED** [review]

File: content/fundamentals/_partials/_what-is-cloudflare.md [hunk 3]
```diff
@@ -5,14 +5,15 @@ _build:
   list: never
 ---
 
-Cloudflare is a global network of [servers](https://www.cloudflare.com/learning/cdn/glossary/edge-server/). When you add your application to Cloudflare, we use this network to sit in between requests and your [origin server](https://www.cloudflare.com/learning/cdn/glossary/origin-server/).
+<div style="position: relative; padding-top: 56.25%;">
+    <iframe
```
Comment (kodster28) [inline content/fundamentals/_partials/_what-is-cloudflare.md]: You should likely move this into a shortcode, similar to https://github.com/cloudflare/cloudflare-docs/blob/production/layouts/shortcodes/youtube.html

**Review by kodster28 — CHANGES_REQUESTED** [review]
Also, look out for the padding under the videos... they seem to need a bit more if there's text immediately after like the `what is Cloudflare` one.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**